py/picca/delta_extraction/data_catalogues/desi_tile.py

```diff
@@ -248,7 +239,7 @@ def read_file(self, filename, catalogue):
         if "cumulative" in self.input_directory:
             select = ((catalogue['TILEID'] == tile_spec) &
                       (catalogue['PETAL_LOC'] == petal_spec) &
-                      (catalogue['LAST_NIGHT'] == night_spec))
+                      (catalogue['LASTNIGHT'] == night_spec))
         else:
             select = ((catalogue['TILEID'] == tile_spec) &
                       (catalogue['PETAL_LOC'] == petal_spec) &
```

Comment:
What would happen if "cumulative" was not declared? It looks like "LASTNIGHT" would not be defined and the code would crash later? 

Do the "daily catalogues" have "cumulative" folders? If so, why do we need the second option?

Reply:
@iprafols - this is still pending

Reply:
Here the usage of LASTNIGHT is unrelated to that of the blinding. 
Here we are not defining column LASTNIGHT. Instead, we are using it to figure out in which file the spectra are saved. If "cumulative" is not declared, then this is not needed as the spectra are not coadded and thus they are saved in the nights they were observed. 
Thus, I think there is nothing to fix here.